packages/fxa-payments-server/src/routes/Product.tsx

```diff
@@ -15,7 +15,7 @@ import {
 import ProductValueProposition from '../components/ProductValueProposition';
 import { useCheckboxState } from '../lib/hooks';
 
-type ProductProps = {
+export type ProductProps = {
```

Comment:
I'm guessing this indicates a bug in the mock setup?

![image](https://user-images.githubusercontent.com/44390/58832750-40234800-8615-11e9-8582-00b2829e1c6e.png)

Reply:
This will be a slight bitrot from other PRs getting merged before this one